pkg/controller/storagecluster/initialization_reconciler.go

```diff
@@ -61,6 +61,23 @@ func (r *ReconcileStorageCluster) ensureStorageClasses(instance *ocsv1.StorageCl
 	return nil
 }
 
+func (r *ReconcileStorageCluster) newOBCStorageClass(initData *ocsv1.StorageCluster) *storagev1.StorageClass {
+	reclaimPolicy := corev1.PersistentVolumeReclaimRetain
```

Comment:
Looks like csi uses `PersistentVolumeReclaimDelete`, let's follow the same pattern

Reply:
Done...